_posts/2025-02-21-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release 2025.09
+author: Priya
+date: 2025-02-21
+category: Jekyll
+layout: post
+---
+### Product Enhancement :
+##### Automation-
+Mammoth introduces new features to enhance the automation creation process. Users are now required to define at least one action and condition, each with configurable details. Multiple actions can be added to an automation for greater flexibility.
+##### Dataset schema- 
+Mammoth introduces a new dataset schema modal, allowing users to easily edit dataset schemas. The modal provides an intuitive interface for managing and modifying schema details, offering greater flexibility and control over dataset configurations.
+##### These are now live on app as an alpha feature. If you'd like to try them out please reach out to us at [support@mamoth.io](mailto:support@mamoth.io)
+
+### Defect Fixes:
+* Resolved an issue where users were unable to save as dataset in draft mode.
+* Fixed a bug where the Convert Column Type function in the column menu displayed invalid column types.
+* Addressed an issue where an empty explore card appeared after converting a column.
```

Comment:
Can we maintain "Resolved an issue" instead of trying different ways to say the same thing?